tools/ci/travis-install.py

```diff
@@ -2,6 +2,7 @@
 import shutil
 import subprocess
 
+MOAB_VERSION= '5.0'
```

Comment:
space before =

Reply:
Removing that variable altogether. MOAB install is done in a different place now.